tests.js

```diff
@@ -64,14 +65,28 @@ const tests = {
     );
     assert.equal(issue, "ENG-1234");
   },
+  testTitlePrefixException: () => {
+    const exception = exceptioncheck.find(
+      "nit",
+      "nit: ENG-1234 Something of value",
+    );
+    assert.equal(exception, "nit");
+  },
+  testTitlePrefixNoException: () => {
+    const exception = exceptioncheck.find(
+      "nit",
+      "feat: ENG-1234 Something of value",
+    );
+    assert.equal(exception, undefined);
+  },
```

Comment:
LGTM, I would just add one more test where `findIssue` calls `exceptioncheck` and properly skips it (instead of only tests of the exception check directly)

Reply:
do you mean a check of the top level `run()` function?

Reply:
I think we'd have to refactor the top-level run to call into a helper function that injects its import dependencies that we unit test. Unless there's a way to directly mock imports that I'm not seeing. I can address this in a separate PR 🙏